pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -575,36 +582,19 @@ func TestAuthnFilterConfig(t *testing.T) {
 	jwksURI := ms.URL + "/oauth2/v3/certs"
 
 	cases := []testCase{{
-		name: "no-request-authn-rule",
-		expected: &http_conn.HttpFilter{
-			Name: "istio_authn",
-			ConfigType: &http_conn.HttpFilter_Config{
-				Config: pilotutil.MessageToStruct(&authn_filter.FilterConfig{
-					Policy: &authn_alpha.Policy{
-						Peers: []*authn_alpha.PeerAuthenticationMethod{
-							{
-								Params: &authn_alpha.PeerAuthenticationMethod_Mtls{
-									Mtls: &authn_alpha.MutualTls{},
-								},
-							},
-						},
-						PeerIsOptional:   true,
-						OriginIsOptional: true,
-						PrincipalBinding: authn_alpha.PrincipalBinding_USE_ORIGIN,
-					},
-				}),
-			},
-		},
+		name:     "no-request-authn-rule",
+		expected: nil,
 	}, {
-		name: "single-request-authn-rule",
-		in: []*model.Config{
-			{
-				Spec: &v1beta1.RequestAuthentication{
-					JwtRules: []*v1beta1.JWT{
-						{
-							Issuer:  "https://secret.foo.com",
-							JwksUri: jwksURI,
-						},
+		name: "no-request-authn-rule-alphafallback",
```

Comment:
to be complete, can you add a test case when there are no request authn (both old and new, but have mTLS in alpha policy)

Reply:
done.
only-mtls-alpha-fallback